web/src/user_search.ts

```diff
@@ -12,7 +12,7 @@ export class UserSearch {
     // details of populating the list when we change.
 
     $widget = $("#user_search_section").expectOne();
-    $input = $<HTMLInputElement>("input.user-list-filter").expectOne();
+    $input = $<HTMLInputElement>("input.people-list-filter").expectOne();
```

Comment:
What if we did `buddy-list-filter`? I feel like that aligns with the newer `buddy_data` / `buddy_list` names we're using.